src/components/Chat/BottomSheetGif.js

```diff
@@ -10,6 +10,7 @@ import {
   FlatList
 } from 'react-native';
 import FastImage from 'react-native-fast-image';
+import {debounce} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `debounce` from lodash is a good addition. It helps to limit the rate at which a function can fire, which in this case is useful for limiting the number of API calls when searching for GIFs.
```diff
+ import {debounce} from 'lodash';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->